server/modules/nodeDef/manager/nodeDefManager.js

```diff
@@ -88,14 +86,16 @@ const updateNodeDefProps = async (user, surveyId, nodeDefUuid, props, propsAdvan
       ? await _updateNodeDefOnCyclesUpdate(surveyId, nodeDefUuid, props[NodeDef.propKeys.cycles], t)
       : []
 
+    const nodeDef = await NodeDefRepository.updateNodeDefProps(surveyId, nodeDefUuid, props, propsAdvanced, t)
+
     const logContent = {
       uuid: nodeDefUuid,
       ...(R.isEmpty(props) ? {} : { props }),
-      ...(R.isEmpty(propsAdvanced) ? {} : { propsAdvanced })
+      ...(R.isEmpty(propsAdvanced) ? {} : { propsAdvanced }),
+      keys: Object.keys({ ...props, ...propsAdvanced }),
```

Comment:
why are you storing the keys? aren't them all keys except uuid?

Reply:
right, I can extract the keys later on in the MessageParser